src/pages/ContentScripts/features/repo-sidebar-labels/OpenDiggerLabel.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { CommonMeta } from '../../../../api/common';
+
+interface OpenDiggerLabelProps {
+  label: CommonMeta['labels'][0];
+}
+
+const OpenDiggerLabel: React.FC<OpenDiggerLabelProps> = ({ label }) => {
+  return (
+    <a
+      id={`opendigger-label-${label.id}`}
+      className="topic-tag topic-tag-link"
+      style={{
+        display: 'inline-flex',
+        alignItems: 'center',
+        paddingLeft: '4px',
+      }}
+      href="https://open-digger.cn"
+      target="_blank"
+    >
+      <img
+        style={{
+          marginRight: '4px',
+          borderRadius: '50%',
+        }}
+        src="https://open-digger.cn/img/logo/logo-blue-round-corner.png"
```

Comment:
BTW, I create a new small logo for HyperCRX to use, this is a 36px logo, https://open-digger.cn/img/logo/logo-blue-round-corner-36px.png .

This will definitely reduce the network and save some cost for OSS.

Reply:
No problem, I will change for the smaller one.